VocalRaidAssistant.lua

```diff
@@ -69,7 +69,8 @@ function VRA:OnInitialize()
           tooltip:Show()
      end,
 	})
-	VRA.ICON:Register("MyLDB", MyLDB, self.db.profile.minimapIconTable)
+	if not VRALDBIconDB then VRALDBIconDB = {} end
+	VRA.ICON:Register(addonName, MyLDB, VRALDBIconDB)
```

Comment:
No need to create an own DB here, we can store this info in existing db.
VRA.ICON:Register(addonName, MyLDB, profile.general.minimap)
Just add empty minimap to defaults

Reply:
I did this initially but every other addon seemed to have a separate DB for the minimap. Maybe just because of copy/paste implementation from an empty project?
Fixed